vm/src/frame.rs

```diff
@@ -840,12 +840,25 @@ impl ExecutingFrame<'_> {
             }
             bytecode::Instruction::SetupWith { end } => {
                 let context_manager = self.pop_value();
-                let enter_res = vm.call_special_method(
-                    context_manager.clone(),
-                    identifier!(vm, __enter__),
-                    (),
-                )?;
-                let exit = context_manager.get_attr(identifier!(vm, __exit__), vm)?;
+                let error_string = || -> String {
+                    format!(
+                        "'{:.*}' object does not support the context manager protocol",
+                        200,
+                        context_manager.class().name(),
+                    )
```

Comment:
```suggestion
                    format!(
                        "'{:200}' object does not support the context manager protocol",
                        context_manager.class().name(),
                    )
```
is this the same thing?